### ISSUE TITLE
AutoConnector needs close and wait_closed

### ISSUE THREAD

**Issue by tkram01** [aio-libs/aioredis#135]
Calling close and wait_closed on Redis generates an error since these are not implemented. It also can make it hard to exit gracefully since you can't wait for tasks to finish easily.

`````` python
@@ -46,15 +46,6 @@
                     self._conn = conn
         return self._conn

-    def close(self):
-        if self._conn is not None and not self._conn.closed:
-            self._conn.close()
-
-    @asyncio.coroutine
-    def wait_closed(self):
-        if self._conn is not None and not self._conn.closed:
-            yield from self._conn.wait_closed()
-

 class Redis(GenericCommandsMixin, StringCommandsMixin,
             HyperLogLogCommandsMixin, SetCommandsMixin,```
``````

**Comment by popravich** [comment]
Thanks for report, will fix this.

**Comment by tkram01** [comment]
I see you made a branch for this. Can you merge it in? Is there anything I can do to get this integrated?

**Comment by popravich** [comment]
Hi, actually AutoConnector is deprecated and will be removed.
Pool is now used inside Redis so there is no need in AutoConnector.
The new code is in master but not yet released.

**Comment by popravich** [comment]
Marking this as "Won't fix" in favor of v1.0 release.
